src/panels/todo/ha-panel-todo.ts

```diff
@@ -148,6 +148,9 @@ class PanelTodo extends LitElement {
           @click=${this._handleEntityPicked}
           .entityId=${list.entity_id}
           .activated=${list.entity_id === this._entityId}
+          style=${!showPane && !this.mobile
+            ? "max-width: calc(100vw - 120px)"
+            : ""}
```

Comment:
Shouldn't be a class instead of inline style?

Reply:
yeah